app/src/org/commcare/utils/ViewUtils.java

```diff
@@ -25,4 +27,9 @@ public static void showSnackBarWithDismissAction(View view, String message) {
         snackbar.setAction(view.getContext().getString(R.string.ok),v -> snackbar.dismiss());
         snackbar.show();
     }
+
+    public static int getNotificationIcon(Context context) {
```

Comment:
don't think this is the right class for this to go, think `NotificationUtil` would be a better choice here.